.github/workflows/publish.yaml

```diff
@@ -19,4 +19,11 @@ jobs:
       with:
         snap: ${{ steps.build.outputs.snap }}
         release: latest/edge
+    - name: Upload charmcraft logs
+      uses: actions/upload-artifact@v2
```

Comment:
I think v3 is available

Reply:
Bumped in [1283b0b](https://github.com/canonical/charmed-kafka-snap/pull/16/commits/1283b0beabec538c20cdea00c196111f3f61dc26)